internal/characters/xiao/asc.go

```diff
@@ -0,0 +1,25 @@
+package xiao
+
+import (
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/player/character"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+const a1Key = "xiao-a1"
+
+func (c *char) a1() {
+	m := make([]float64, attributes.EndStatType)
+	c.AddAttackMod(character.AttackMod{
```

Comment:
you aren't doing checks using atk and t, so no need to make this an AttackMod. should be a statmod instead